### PR TITLE
Removed Irrelevant Languages from Test Coverage Generation Section

### PR DESCRIPTION
**Pull request by starvader13**
# Pull Request Template

## Description

Refactored the documentation to remove irrelevant language-specific instructions from the test coverage generation section. 

###Refactored Documentation:

1. Removed Java-related instructions from the other code sections.
2. Ensured other sections do not include a Go-specific Graceful Shutdown.
3. Removed any unrelated code snippets or instructions from the sections specific to each programming language.
4. Make sure that the different cleanup function is only included in the corresponding languages.

Fixes https://github.com/keploy/keploy/issues/2070

## Type of change.

- [x] This change requires a documentation update.

### CODE REVIEW

**Comment by starvader13** [comment]
@shivamsouravjha Review Required!!

File: versioned_docs/version-2.0.0/server/sdk-installation/go.md [hunk 1]
```diff
@@ -22,6 +22,45 @@ import WhatAreKeployFeatures from './index.md'
...
+	r.POST("/url", putURL)
+	// should be called before starting the API server from main()
+	GracefulShutdown()
+
+	r.Run()
+}
+```
+
+
 ## Usage
```
Comment (shivamsouravjha) [inline versioned_docs/version-2.0.0/server/sdk-installation/go.md:65]: seems like the note of jvm would still remain in this.

**Comment by shivamsouravjha** [comment]
also make sure the note of jvm not be present in go and js similary go code not be present in java or go.

File: versioned_docs/version-2.0.0/server/sdk-installation/go.md (hunk 1 above)
Comment (starvader13) [reply versioned_docs/version-2.0.0/server/sdk-installation/go.md:65]: @shivamsouravjha  I'm not quite sure which note you're referring to, as each language section appears to be dedicated solely to its respective language. Can you please guide me to the actual issue?

**Comment by starvader13** [comment]
> Also make sure the note of jvm not be present in go and js similary go code not be present in java or go.

Same for this one. There is no interference of any language but for a better description let me share you with the structure.

### Index
- **Prerequisites**: General instructions applicable to all languages.
- **Usage Instructions**: Keploy-specific commands, not language-specific.
- **File Structure**: General structure applicable to any Keploy-supported language.
- **Conclusion**: General and applicable to all languages without interference.

### Go
- **Prerequisites**: Specific to Go.
- **Usage Instructions**: Contains Go commands and scripts.
- **Code Examples**: Written in Go.
- **Conclusion**: Exclusively Go-related content.

### Java
- **Prerequisites**: Specific to Java.
- **Usage Instructions**: Involves Maven and Java-specific commands.
- **Code Examples**: XML configuration for Maven.
- **Conclusion**: Exclusively Java-related content.

### JavaScript
- **Prerequisites**: Specific to JavaScript.
- **Usage Instructions**: Uses npm and JavaScript-specific commands.
- **Code Examples**: JSON for `package.json`.
- **Conclusion**: Exclusively JavaScript-related content.

### Python
- **Prerequisites**: Specific to Python.
- **Usage Instructions**: Uses Python commands and scripts.
- **Code Examples**: Written in Python.
- **Conclusion**: Exclusively Python-related content.

File: versioned_docs/version-2.0.0/server/sdk-installation/go.md (hunk 1 above)
Comment (shivamsouravjha) [reply versioned_docs/version-2.0.0/server/sdk-installation/go.md:65]: image
here this is a go code still it has java code.

In java there is go code 
image

In js it has go and java code


image

Comment (starvader13) [reply versioned_docs/version-2.0.0/server/sdk-installation/go.md:65]: I have made all the changes that you are referring to here, Wait let me share what my local build contains.

# Go
image

# Java
image

# Javascript
image

# Python
image

**Comment by shivamsouravjha** [comment]
the changes look good can you resolve the conflicts and the build failures?

**Comment by starvader13** [comment]
> The changes look good can you resolve the conflicts and the build failures?

Roger that

**Comment by starvader13** [comment]
@shivamsouravjha I don't know why Prettier is failing. Can you look into that?

**Review by shivamsouravjha — APPROVED** [review]
LGTM!

**Pull request merged**